tools/build_velox.sh

```diff
@@ -9,8 +9,8 @@ COMPILE_VELOX=OFF
 VELOX_HOME=${3:-/root/velox}
 ENABLE_EP_CACHE=OFF
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/jinchengchenghh/velox.git
```

Comment:
does is need to modify back to `oap-project` repo?

Reply:
Yes, I asked @rui-mo to review velox PR https://github.com/oap-project/velox/pull/50. After it merged, I will change to oap-project and merge this one

Reply:
Merged the PR for Velox.